js/api/Triggers.js

```diff
@@ -14,15 +14,11 @@ export function checkSurveyTimeTriggers(survey, omitNotifications) {
   const triggers = realm.objects('TimeTrigger').filtered(`surveyId="${survey.id}" AND triggered == false`);
   const now = new Date();
 
-  // Make the expiration date 90 days
-  let past = new Date();
-  past = past.setDate(past.getDate() - 90);
-
   // Record the new trigger
   const triggerLength = triggers.length;
   realm.write(() => {
     for (let i = 0; i < triggerLength; i++) {
-      if (triggers[i] && triggers[i].datetime < now && triggers[i].datetime > past) {
+      if (triggers[i] && triggers[i].triggered === false && triggers[i].datetime < now) {
```

Comment:
That check was a development placeholder before we added the ability to de-activate surveys. That logic didn't accomplish anything now aside from disabling timetriggers without notice.